app/helpers/course_helper.rb

```diff
@@ -5,15 +5,17 @@ def course_manage_error_link(field, error)
                 href: enrichment_error_url(
                   provider_code: @provider.provider_code,
                   course: @course,
-                  field: field
+                  field: field.to_s
                 )
   end
 
   def course_summary_label(key, field)
     if @errors&.key? field
-      error = @errors[field]
       content_tag :dt, class: 'govuk-summary-list__key course-parts__fields__label--error' do
-        content_tag(:span, key) + course_manage_error_link(field, error)
+        [
+          content_tag(:span, key),
+          *@errors[field].map { |error| course_manage_error_link(field, error) }
```

Comment:
What does the `*` do here?

Reply:
It's the [Ruby splat operator](https://www.honeybadger.io/blog/ruby-splat-array-manipulation-destructuring/), which is similar to the JavaScript spread operator `...`, which depending on the context will do a few things but mostly means "take this array / hash and turn it into individual things".

Reply:
This code is generating an array of error links and then joining them up into an array with the above `content_tag`, which then get added up together using `.reduce(:+)`.